src/components/Menu.tsx

```diff
@@ -76,7 +76,24 @@ const Menu = (): JSX.Element => {
     fetchUserSubmissions(dispatch, state.userList.handles, wait);
   };
 
+  //codeforces username can contain only alpanumeric characters, underscore, hyphen and length can be between [3, 24] only
+  const isValidCodeforcesUsername = (username) => {
+    const regex = /^[a-zA-Z0-9_-]{3,24}$/;
```

Comment:
Can you show any source backing this logic?

Reply:
There is a observation if you make(codeforces username rules, it will not allow you to have username which are not following these rules) , i was trying to find an article/blog on CF that supports that regex, but its no where written specifically but on Merlin and GPT i could find such those rules. sharing the screen shot if that can be help, if it's not convincing we will need to stick to just check for empty string only and removal of additional spaces.
<img width="1035" alt="Screenshot 2024-06-25 at 9 27 09 PM" src="https://github.com/mbashem/cftracker/assets/96096248/d7c73708-5092-453d-98b6-d2671be25c8f">

Reply:
Sorry, It is not convincing. Please remove this.